styles/style.css

```diff
@@ -0,0 +1,159 @@
+@import url('https://fonts.googleapis.com/css2?family=Tangerine:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Charm:wght@400;700&family=Tangerine:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Caudex:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+:root {
+
+    font-size: 16px;
+    /* CSS HEX */
+    --silver: #a5a19d;
+    --silver-2: #C6C0B6ff;
+    --chocolate-cosmos: #5B1C1Dff;
+    --field-drab: #65512Cff;
+    --white-smoke: #F3F3F3ff;
+
+}
+
+* {
+    /* border: 1px dotted blue; */
+}
+
+h1 {
+    font-family: "Tangerine", cursive;
+    font-weight: 400;
+    font-style: normal;
+}
+
+* {
+    font-family: "Caudex", serif;
+    font-weight: 400;
+    font-style: normal;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: var(--white-smoke);
+    color: #333;
+    height: 100vh;
+    display: grid;
+    grid-template-rows: 1fr 6fr .5fr;
+
```

Comment:
Small style/organization note that applies to all the files: I see inconsistent spacing across the style files, I suggest removing any blank lines inside rule sets and keep a single blank line between each rule set to separate them.